simapp/simd/cmd/genaccounts.go

```diff
@@ -140,7 +141,7 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 			}
 			authGenState.Accounts = genAccs
 
-			authGenStateBz, err := cdc.MarshalJSON(authGenState)
+			authGenStateBz, err := json.MarshalIndent(authGenState, "", "")
```

Comment:
should we add a test for this command?

In #7037, I added a test in `simapp/simd/cmd/cmd_test.go`. But maybe they are not perfect (ref: https://github.com/cosmos/cosmos-sdk/pull/7037#discussion_r471322433). Were do you think they should go @alexanderbez?

Reply:
Ideally, they're tested where they're defined. We can test this command in this package as you've done.